src/test/suite/fixtures/recorded/languages/elm/changeKey.yml

```diff
@@ -0,0 +1,30 @@
+languageId: elm
+command:
+  spokenForm: change key
+  version: 2
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: collectionKey}
+  usePrePhraseSnapshot: true
+  action: {name: clearAndSetSelection}
+initialState:
+  documentContents: |
+    f : { x : Int }
```

Comment:
Should support "type" for `Int` here as well.  Maybe you already do, but either way a test for that would be nice 😊

Reply:
If the cursor is on `x`, should `type` select `Int` (the type of `x`) or `{ x : Int }` (the closest enclosing type)? 🤔 
Currently it does the latter

Reply:
Should be the former (ie `Int`)